pkg/test/resourcefixture/testdata/basic/alloydb/v1beta1/alloydbcluster/fullalloydbcluster/create.yaml

```diff
@@ -51,3 +51,12 @@ spec:
   encryptionConfig:
       kmsKeyNameRef: 
         name: kmscryptokey-${uniqueId}
+
```

Comment:
Remove the newline? We don't add space in between lines of the same object in the YAML configuration. This YAML file is not following a best practice.

Reply:
Done